lib/posix-process/clone.c

```diff
@@ -310,19 +310,19 @@ static void _clone_child_gc(struct uk_thread *t)
 /*
  * NOTE: The clone system call and the handling of the TLS
  *
- *       `_clone()` assumes that a passed TLS pointer is an Unikraft TLS.
- *       The only exception exists if `_clone()` is called from a context
+ *       `uk_clone()` assumes that a passed TLS pointer is an Unikraft TLS.
+ *       The only exception exists if `uk_clone()` is called from a context
  *       where a custom TLS is already active (depends on
  *       `CONFIG_LIBSYSCALL_SHIM_HANDLER_ULTLS`). In such a case, an
  *       Unikraft TLS is allocated but the passed TLS pointer is activated.
  *       The reason is that Unikraft libraries place TLS variables and use
  *       the TLS effectively as TCB.
- *       In case no TLS is handed over (CLONE_SETTLS is not set), _clone will
+ *       In case no TLS is handed over (CLONE_SETTLS is not set), uk_clone will
  *       still allocate an Unikraft TLS but sets the TLS architecture pointer
  *       to zero.
  */
-static int _clone(struct clone_args *cl_args, size_t cl_args_len,
-		  struct ukarch_execenv *execenv)
+int uk_clone(struct clone_args *cl_args, size_t cl_args_len,
```

Comment:
`__sz`?

Reply:
Had I written this from scratch yes, but since the rest of this file uses `size_t` I chose to stick to the existing convention.